x-pack/plugins/apm/public/components/app/service_overview/service_overview.test.tsx

```diff
@@ -28,11 +28,27 @@ import {
   getCallApmApiSpy,
   getCreateCallApmApiSpy,
 } from '../../../services/rest/callApmApiSpy';
+import { fromQuery } from '../../shared/Links/url_helpers';
+import { MockUrlParamsContextProvider } from '../../../context/url_params_context/mock_url_params_context_provider';
+import { uiSettingsServiceMock } from '../../../../../../../src/core/public/mocks';
 
 const KibanaReactContext = createKibanaReactContext({
   usageCollection: { reportUiCounter: () => {} },
 } as Partial<CoreStart>);
 
+const mockParams = {
+  rangeFrom: 'now-15m',
+  rangeTo: 'now',
+  latencyAggregationType: LatencyAggregationType.avg,
+};
+
+const location = {
+  pathname: '/services/test%20service%20name/overview',
+  search: fromQuery(mockParams),
+};
+
+const uiSettings = uiSettingsServiceMock.create().setup({} as any);
```

Comment:
TIL these exist. Some time soon I'll go and get us set up to use these automatically in Jest and Storybook.